lib/l10n/app_en.arb

```diff
@@ -71,6 +71,7 @@
   "two_fa_desc": "Please enter your confirmation code below",
   "two_fa_select": "Select option {number}",
   "two_fa_email": "Email code",
+  "two_fa_email_address": "Email address",
```

Comment:
what do you think declaring the key name as `common_email_address` ?

Reply:
Updated.